packages/lesswrong/server/scripts/arbitalImport/arbitalImport.ts

```diff
@@ -1341,3 +1345,187 @@ Globals.hideContentFromNonDefaultDomains = async (mysqlConnectionString: string)
   });
 }
 
+async function importPagePairs(
+  database: WholeArbitalDatabase,
+  resolverContext: ResolverContext,
+  options: ArbitalImportOptions
+): Promise<void> {
+  const conversionContext = await buildConversionContext(database, options);
+  const pageInfosById = conversionContext.pageInfosByPageId;
+
+  console.log("Truncating ArbitalTagContentRels collection");
+  await ArbitalTagContentRels.rawRemove({});
+
+  console.log("Importing page pairs");
+
+  const filteredPagePairs = database.pagePairs.filter(pair => {
+    const { parentId, childId, type } = pair;
+    const validType = ['tag', 'subject', 'requirement'].includes(type);
+    const parentInfo = pageInfosById[parentId];
+    const childInfo = pageInfosById[childId];
+    const pagesExist = parentInfo && childInfo;
+    const pagesNotDeleted = pagesExist && !parentInfo.isDeleted && !childInfo.isDeleted;
+    return validType && pagesExist && pagesNotDeleted;
+  });
+
+  console.log(`Importing ${filteredPagePairs.length} page pairs`);
+
+  for (const pair of filteredPagePairs) {
+    const { parentId, childId, type, level, isStrong, createdAt } = pair;
+
+    // console.log(`Importing page pair ${pair.id}`);
+
+
+    // Map Arbital page IDs to our internal tag IDs
+    const [parentMultiDoc, childMultiDoc, parentTag, childTag] = await Promise.all([
+      MultiDocuments.findOne({ 'legacyData.arbitalLensId': parentId, fieldName: 'description' }),
+      MultiDocuments.findOne({ 'legacyData.arbitalLensId': childId, fieldName: 'description' }),
```

Comment:
I... don't expect this to work correctly, since there'll be multiple copies of each lens and you won't know which one corresponds to a non-deleted tag.

Reply:
Hmmmm